_posts/2017-03-23-alexa-trivia-skill.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+title: My First Alexa Trivia Skill
+feature-img: "img/min-back.jpg"
+---
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Whenever learning a new coding skill the documentation always seem scattered or as if there is a million and one ways to do something. Now I know there multiple solutions to every problem but when starting out it can be overwhelming. When starting the learn about writing an Alexa trivia skill, it was probably the easiest coding endeavor I have done. The documentation was clean and clear.  Now of course every learning process is not clear of any hiccups. Made my only mistake by writing my APP_ID variable as a string, but that was my own mistake since it was in the documentation.  I ended up writing a trivia skill called WoW Trivia. It is a short trivia skill with about twenty trivia questions but it was a good starting point.  
```

Comment:
Some grammar suggestions:

"Whenever learning a new coding skill, the documentation can feel scattered or unclear on how to get started. Looking back, I know there can multiple solutions to every problem but starting out can be overwhelming. 

During my Bloc apprenticeship, I chose to write an Alexa trivia skill. It was the easiest coding endeavor I have done. The Alexa documentation was clean and clear.  Now, of course, every learning process is not clear of any hiccups. I made my only mistake by writing my APP_ID variable as a string, but that was my own oversight since it was in the documentation.  I ended up writing a trivia skill called WoW Trivia. It is a short trivia skill with about twenty trivia questions but it was a good starting point.

If you have any interest in Alexa development, I suggest you do..."